core/biogeophys/cable_common.F90

```diff
@@ -684,6 +685,105 @@ subroutine get_unit(iunit)
 
   end subroutine get_unit
 
+  FUNCTION latitude_dimid(FileID)
+    !*## Purpose
+    !
+    ! Find the id of the latitude dimension
+    !
+    !## Method
+    !
+    ! Run through a list of possible names for the latitude dimension, until one
+    ! is found in the NetCDF dimensions.
+
+    INTEGER :: FileID, latitude_dimid
+    INTEGER :: i, ok
```

Comment:
It's a small thing, however if the length of `LatNames` is 0, then `ok` is undefined. Could initialize `ok` for higher guarantees

Reply:
Ah yep, I guess makes sense as a safety thing, if when coupled with the prior change, someone else tries to hijack it for their own purposes and pass an invalid array.